apps/site/app/[locale]/page.tsx

```diff
@@ -92,7 +92,7 @@ const getPage: FC<DynamicParams> = async props => {
   // Gets the current full pathname for a given path
   const pathname = dynamicRouter.getPathname(path);
 
-  const staticGeneratedLayout = DYNAMIC_ROUTES.get(pathname);
+  const staticGeneratedLayout = (await DYNAMIC_ROUTES()).get(pathname);
```

Comment:
One line for await and another for function call. Also if this DYNAMIC_ROUTES is not a constant anymore, but an async method, it's name should be changed and it should be moved to another file.

Reply:
change reverted, no longer needed (thanks to work done on the adapter 🙂) 🥳